src/components/Main/main.scss

```diff
@@ -0,0 +1,20 @@
+.card {
+    padding:20px;
+    margin-bottom:30px !important;
+    width:315px !important;
+    height:360px !important;
+    border:none;
+    flex-direction: column;
+    justify-content:space-between;
+    align-items: center;
+    border-radius:0;
+}
+
+.card>img{
+    object-fit:contain;
+}
+
+.card-body{
```

Comment:
Please follow BEM convention.

Reply:
this 'card-body' is bootstrap's class. Should I add 'card__body' after that ?! Like :
![image](https://user-images.githubusercontent.com/62364608/160055359-4ff771b1-3067-470b-9be4-deba484ed9e6.png)

Reply:
Yes. If wanna overwrite the style of bootstrap, we should create new class,